Documentation/ABI/JSON.md

```diff
@@ -210,7 +210,22 @@ sufficient information to display the event in a human-readable format.
   "fail" | "difference" | "warning" | "details"
 ```
 
+<!--
+  TODO: Publicize issue severity.
+<issue-severity> ::= "warning" | "error" ; additional severity levels may be
+                                         ; added in the future
+-->
+
 <!--
   ["testID": <test-id>,
     ["testCase": <test-case>]]
 -->
+
+## Version history
```

Comment:
I was actually planning to rename the existing file "ABIv0.md" or similar and create a new one to cover ABIv1. I was suggesting we sit down, write a list of what we want to do for v1, and draft it in an entirely separate PR.

Reply:
I've broken out the "pending changes" into a standalone document, with a clear indication that everything there is tentative.

Reply:
Now in #946